test/evergreen.yml

```diff
@@ -5837,7 +5841,6 @@ buildvariants:
     ENABLE_TCMALLOC: 1
     num_jobs: $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
   tasks:
-    - name: ".lint"
```

Comment:
Why did you remove this?

Reply:
I remember seeing a WT ticket that aims to only have one lint occur in a patch build. Follow here: [WT-10982](https://jira.mongodb.org/browse/WT-10982)